### DISCUSSION TITLE
How to Remove or Align this toolbar?

### DISCUSSION THREAD

**Discussion by Ser4ph4** [Godiesc/firefox-gx#169] [Q&A] [answered]
Hello everyone, I was editing some things in my theme in Firefox but I couldn't remove or align this menu (Image below) until I managed to remove it, but I also ended up removing the Close/Minimize/Maximize button... can anyone help me with this? 

![Firefox](https://github.com/Godiesc/firefox-gx/assets/78483475/63c23daf-07e9-45ea-8fea-45cff2a303c0)

**Comment by Godiesc** [comment]
hi,
I guess you are using some tree-tabs extension, for that you have this: https://github.com/Godiesc/firefox-gx/tree/main/Extras/TreeStyle-tabs, are you followed the step to apply that config? maybe is not necessary do it for your own when this theme has the way to do it.

**Reply by Ser4ph4** [reply]
**I think I did something wrong in the code below, can you check?**
a im use Sidebery 


`/* Show sidebar only when the cursor is over it  */
/* The border controlling sidebar width will be removed so you'll need to modify these values to change width */

#sidebar-box{
    --uc-sidebar-width: 38px;
    --uc-sidebar-hover-width: 250px;
    --uc-autohide-sidebar-delay: 200ms; /* Wait 0.6s before hiding sidebar */
    position: relative;
    min-width: var(--uc-sidebar-width) !important;
    width: var(--uc-sidebar-width) !important;
    max-width: var(--uc-sidebar-width) !important;
    z-index:1;
  }
  #sidebar-box[positionend]{ direction: rtl }
  #sidebar-box[positionend] > *{ direction: ltr }
  
  #sidebar-box[positionend]:-moz-locale-dir(rtl){ direction: ltr }
  #sidebar-box[positionend]:-moz-locale-dir(rtl) > *{ direction: rtl }
  
  #main-window[sizemode="fullscreen"] #sidebar-box{ --uc-sidebar-width: 1px; }
  
  #sidebar-splitter{ display: none }
  
  #sidebar-header{
    overflow: hidden;
    color: var(--chrome-color, inherit) !important;
    padding-inline: 0 !important;
  }
  
  #sidebar-header::before,
  #sidebar-header::after{
    content: "";
    display: flex;
    padding-left: 8px;
  }
  
  #sidebar-header,
  #sidebar{
    transition: min-width 250ms linear var(--uc-autohide-sidebar-delay) !important;
    min-width: var(--uc-sidebar-width) !important;
    will-change: min-width;
  }
  #sidebar-box:hover > #sidebar-header,
  #sidebar-box:hover > #sidebar{
    min-width: var(--uc-sidebar-hover-width) !important;
    transition-delay: 0ms !important;
  }
  
  .sidebar-panel{
    background-color: transparent !important;
    color: var(--newtab-text-primary-color) !important;
  }
  
  #TabsToolbar {
    visibility: collapse;
  }
  
  #urlbar:not([focused]) #urlbar-input{ text-align: center !important; }
  -------------
 
`

**Reply by Ser4ph4** [reply]
> hi, I guess you are using some tree-tabs extension, for that you have this: https://github.com/Godiesc/firefox-gx/tree/main/Extras/TreeStyle-tabs, are you followed the step to apply that config? maybe is not necessary do it for your own when this theme has the way to do it.

-------------
**When I disable the option it will also remove the Close/Maximize/Minimize buttons**
![noname](https://github.com/Godiesc/firefox-gx/assets/78483475/dc73113c-91fc-4bed-a024-798c566ecd8d)

**Reply by Godiesc** [reply]
well, I can't help with your personal code into sideberry, I just can help you with the Firefox-GX skin, maybe the place where you found that sideberry's code could help you.

**Reply by Godiesc** [reply]
Here I show how to apply the tree-tab style in about:config page, you can see  the window controls stay:

https://github.com/Godiesc/firefox-gx/assets/22057609/cf43621d-312a-48cf-9f95-90216403e8a7

**Reply by Ser4ph4** [reply, marked as answer]
> Here I show how to apply the tree-tab style in about:config page, you can see the window controls stay:
> Apply-Tree-tabs.mp4

Thank you very much friend, it worked, that was exactly it, I downloaded the theme and redid the settings. Now it's how I wanted it.

**_Much Success and Peace to you_**
**Sincerely** 👍 💯

**Reply by Godiesc** [reply]
I see in the code you shared that you have some code that affect the tabs-bar and for that you have that problem, delete it and test it again and that changes that affect sideberry maybe works like you want.

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/408ed0ef-2fb6-4e2f-8bcc-8975eee632d5)

**Comment by Ser4ph4** [comment]
After the last update of Firefox version 133 the close, minimize and maximize buttons do not work :(
![noname](https://github.com/user-attachments/assets/c456e2f6-2061-4ea2-ab9d-a722cfef8af4)

**Reply by Godiesc** [reply]
hi again,
the theme was tested again and all seems right, be sure your personal code don't make the issue:

PD: I guess you downloaded again the new theme version for Firefox 133.

https://github.com/user-attachments/assets/65a29f5c-2809-4c27-b465-45ced9900709

**Reply by Ser4ph4** [reply]
> hi again, the theme was tested again and all seems right, be sure your personal code don't make the issue:
> 
> PD: I guess you downloaded again the new theme version for Firefox 133.

I'm trying to fix it, how do you remove a border and set the size of that part? I use a direct size in sidebery that seems to be ignored
![noname](https://github.com/user-attachments/assets/30a7a156-b19a-4e79-856e-5bd279d9010f)

**Reply by Godiesc** [reply]
sorry, that style only should be apply for 'tab-center-reborn' extension, soon I will update the [ogx_tree-tabs.css](https://github.com/Godiesc/firefox-gx/blob/main/chrome/components/ogx_tree-tabs.css) to remove the style for other tree-tabs extensions.

**Reply by Godiesc** [reply]
can you test the code, replace the code of `chrome/components/ogx_tree-tabs.css` with this: https://pastebin.com/1s5BnrGF
I deleted the style for other tree-tabs extensions, so you personal code for your extension you should add yourself.

**Reply by Ser4ph4** [reply]
> can you test the code, replace the code of `chrome/components/ogx_tree-tabs.css` with this: https://pastebin.com/1s5BnrGF I deleted the style for other tree-tabs extensions, so you personal code for your extension you should add yourself.

Solved the issue here, now my buttons work. Thank you very much for all the support..
Your efforts are appreciated :)
